pkg/apis/core/helper/helpers.go

```diff
@@ -360,6 +360,13 @@ func ContainsAccessMode(modes []core.PersistentVolumeAccessMode, mode core.Persi
 	return false
 }
 
+func ClaimContainsAllocatedResources(pvc *core.PersistentVolumeClaim) bool {
```

Comment:
Can you reuse (and rename) the existing method? https://github.com/kubernetes/kubernetes/blob/54ad1ad81e90487305a2912e373657696c8cb388/pkg/api/persistentvolumeclaim/util.go#L178-L186